app/src/main/java/com/example/quizacademytask/FlashcardStackFragment.kt

```diff
@@ -86,4 +88,10 @@ class FlashcardStackFragment : Fragment() {
         }
 
     }
+
+    private fun handleTablet() {
```

Comment:
this method does not what name suggests. Either rename to indicate it or split in two - one for checking that device is tablet and one for handling it.